pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -857,6 +857,7 @@ type KumaSDConfig struct {
 	// +required
```

Comment:
```suggestion
	// +kubebuilder:validation:Pattern:="^http(s)?://.+$"
	// +required
```
AFAIK, the server should only support these two schemas. 
CC: @simonpasquier